src/main/webapp/index.jsp

```diff
@@ -11,26 +11,40 @@
   <%@ page import = "com.google.appengine.api.users.UserService" %>
   <%@ page import = "com.google.appengine.api.users.UserServiceFactory" %>
   <% UserService userService = UserServiceFactory.getUserService(); 
-  boolean userLogged = userService.isUserLoggedIn(); %>
-  <body onload="adjustControlBar(<%=userLogged%>)">
+  boolean userLoggedIn = userService.isUserLoggedIn();
+  String categoriesClass = new String();
+  if (userLoggedIn) {
+      categoriesClass = "notFullWidth";
+  } else categoriesClass = "fullWidth";
```

Comment:
This can be neatly expressed with the ternary operator one one line. (https://alvinalexander.com/java/edu/pj/pj010018/)

Reply:
ooooo, this is pretty neat! Thanks, I'll make use of this moving forward :)